eng/native/configuretools.cmake

```diff
@@ -6,7 +6,7 @@ if (CMAKE_C_COMPILER MATCHES "-?[0-9]+(\.[0-9]+)?$")
   set(CLR_CMAKE_COMPILER_FILE_NAME_VERSION "${CMAKE_MATCH_0}")
 endif()
 
-if(NOT WIN32 AND NOT CLR_CMAKE_TARGET_ARCH_WASM)
+if(NOT WIN32 AND NOT CLR_CMAKE_TARGET_BROWSER)
```

Comment:
Note that a lot of this code is targeting all wasm platforms, not just the browser.

Reply:
I agree that at this level `CLR_CMAKE_TARGET_WASM` would be better, we care about browser at libraries (higher) level

Reply:
I disagree, this is looking up tools from the toolchain. `CLR_CMAKE_TARGET_BROWSER` is an explicit alias for Emscripten which only targets Browsers and provides a CMake toolchain file which already defines these variables so we can skip the lookup there.

Other WebAssembly targets might use other toolchains. I'd prefer to keep it like it is.

Reply:
I see, let's keep it this way then